packages/components/checkbox/src/Checkbox.tsx

```diff
@@ -20,41 +21,50 @@ export const Checkbox = forwardRef<HTMLButtonElement, CheckboxProps>(
       disabled,
       onCheckedChange,
       children,
-      ...others
+      ...rest
```

Comment:
We don't have an agreement regarding this name. Maybe should come with agreement ? In most of our components is named `others`